tests/post_training/conftest.py

```diff
@@ -19,6 +19,8 @@ def pytest_addoption(parser):
     parser.addoption("--data", action="store", help="Data directory")
     parser.addoption("--output", action="store", default="./tmp/", help="Directory to store artifacts")
     parser.addoption("--no-eval", action="store_true", help="Skip validation step")
+    parser.addoption("--batch_size", action="store", default=1, type=int, help="Batch size of calibration dataset")
+    parser.addoption("--dynamic_batch_shape", action="store_true", help="Export model with dynamic batch axis")
```

Comment:
Looks like dynamic_batch_shape requared for batch_size > 1, and can be set by batch_size.

Reply:
done